packages/core/src/notifications/types.ts

```diff
@@ -51,9 +51,21 @@ export interface UIRenderInstructions {
         [`en-US`]: {
             title: string
             description: string
+            descriptionPreview?: string // optional property for toast
         }
     }
-    // TODO actions
+    onRecieve: string
+    onClick: {
+        type: string
+        url?: string // optional property for 'openUrl'
+    }
+    buttons?: Array<{
```

Comment:
`uiRenderingInstructions` will be pretty IDE specific (example: we are adding onReceive and onClick to encapsulate possiblities with VSC specifically), but we should try to align to the original schema as much as possible to not cause confusion. Let's keep this as `actions` instead, even though buttons is probably a better term.